testing/tests/DevExpress.localization/localization.utils.tests.js

```diff
@@ -11,13 +11,15 @@ testModule('Localization utils', () => {
         assert.strictEqual(toFixed(4.64, 2), '4.64');
         assert.strictEqual(toFixed(35.855, 2), '35.86');
         assert.strictEqual(toFixed(35.855, 5), '35.85500');
-        assert.strictEqual(toFixed(-4.645, 2), '-4.65');
+        assert.strictEqual(toFixed(-4.645, 2), '-4.64');
```

Comment:
I think it is the wrong result. '-4.65' is correct

Reply:
```
Chrome:
>new Intl.NumberFormat('en-US', { minimumFractionDigits: 2, maximumFractionDigits: 2 }).format(-4.645)
<"-4.65"
But:
>Globalize.locale( "en" );
>Globalize.numberFormatter({ minimumFractionDigits: 2, maximumFractionDigits: 2 })( -4.645 );
<"-4.64"

And more:
IE:
>new Intl.NumberFormat('en-US', { minimumFractionDigits: 2, maximumFractionDigits: 2 }).format(-4.645)
<"-4.64"

```

Reply:
Globalizejs still have a [problem](https://github.com/globalizejs/globalize/issues/884) with inaccurate floating point arithmetic. So you're right '-4.65' is correct

Reply:
Looks like Globalizejs has a [problem](https://github.com/globalizejs/globalize/issues/884) with number rounding, so @alexander-kotov-dx  is correct